### PR TITLE
Skeletonize the project

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request skeletonizes this project from [cisagov/skeleton-python-library].

## 💭 Motivation and context ##

1. This project is in active use and has not been skeletonized.
2. I need to make changes to this project to resolve a bug and I do not want to do so without at least our linting and build verification in place.

## 🧪 Testing ##

Automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Add a tag or create a release.

[cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# This is the configuration for code coverage checks
+# https://coverage.readthedocs.io/en/latest/config.html
+
+[run]
+source = src/example
```
Comment (dav3r) [inline .coveragerc]: Replace skeleton directory name:
```suggestion
source = src/trustymail
```

File: src/trustymail/trustymail.py [hunk 2]
```diff
@@ -459,7 +468,7 @@ def spf_scan(resolver, domain):
 
 def parse_dmarc_report_uri(uri):
     """
-    Parses a DMARC Reporting (i.e. ``rua``/``ruf)`` URI
+    Parse a DMARC Reporting (i.e. ``rua``/``ruf)`` URI.
```
Comment (dav3r) [inline src/trustymail/trustymail.py]: Fix a typo while we are here:
```suggestion
    Parse a DMARC Reporting (i.e. ``rua``/``ruf``) URI.
```

**Review by dav3r — APPROVED** [review]
Looks great aside from the two minor items I noted.  Strong work! 🚀

File: .coveragerc (hunk 1 above)
Comment (mcdonnnj) [reply .coveragerc]: Ack I thought I'd changed that.

**Review by jsf9k — APPROVED** [review]

**Pull request merged**